test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj

```diff
@@ -1,21 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
-    <RuntimeFrameworkVersion>1.0.5</RuntimeFrameworkVersion>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="1.0.*" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.WebSockets.Server" Version="0.1.*" />
-    <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="1.0.*" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="1.0.*" />
-    <PackageReference Include="Microsoft.AspNetCore.Server.WebListener" Version="1.0.*" />
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Https" Version="1.0.*" />
-    <PackageReference Include="Microsoft.AspNetCore.ResponseCompression" Version="1.0.*" />
-    <PackageReference Include="Microsoft.AspNetCore.ResponseCaching" Version="1.0.*" />
-    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.0.*" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.CommandLine" Version="1.0.*" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.0.*" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
```

Comment:
testwebsite.cs has a hard-coded runtime version of netcoreapp1.0.
So, please update that with netcoreapp2.0 together. 

https://github.com/aspnet/AspNetCoreModule/blob/5c2906a7c6156e9011690679c617502c119b8eee/test/AspNetCoreModule.Test/Framework/TestWebSite.cs#L201